src/rard/research/forms.py

```diff
@@ -572,9 +572,9 @@ def __init__(self, *args, **kwargs):
         if original_text:
             self.fields["content"].widget.attrs["data-object"] = original_text.pk
             # Only enable apparatus criticus editing if object exists
-            self.fields["content"].widget.attrs[
-                "class"
-            ] = "enableApparatusCriticus enableCKEditor"
+            self.fields["content"].widget.attrs["class"] = "enableCKEditor"
+            if original_text.apparatus_criticus_items.count() > 0:
+                self.fields["content"].widget.attrs["class"] = "enableApparatusCriticus"
```

Comment:
I haven't tested this in practice, but won't this set the class to "enableApparatusCriticus" rather than "enableCKEditor enableApparatusCriticus" if there any apparatus criticus items? If so, I'd expect the editor not to get initialised. Likewise for OriginalTextForm below.